source/vocab/display.jsonld

```diff
@@ -615,7 +614,8 @@
             "illustrativeContent",
             "cartographicAttributes",
             "isPartOf",
-            {"inverseOf": "instanceOf"}
+            {"inverseOf": "instanceOf"},
+            "translationOf"
```

Comment:
Note that this will place {"inverseOf": "instanceOf"} in the "main form" instead of in the "incoming links" section.
Is that intended?

Reply:
[97217ab](https://github.com/libris/definitions/pull/419/commits/97217ab0a9e899f9b130a238aeb253124d8baf42)